onnxruntime/test/onnx/main.cc

```diff
@@ -310,19 +310,6 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
 #ifdef USE_OPENVINO
       //Setting default optimization level for OpenVINO can be overriden with -o option
       sf.SetGraphOptimizationLevel(ORT_DISABLE_ALL);
-      if (p_models != 1) {
-        fprintf(stderr, "OpenVINO doesn't support more than 1 model running simultaneously default value of 1 will be set \n");
-        p_models = 1;
-      }
-      if (concurrent_session_runs != 1) {
-        fprintf(stderr, "OpenVINO doesn't support more than 1 session running simultaneously default value of 1 will be set \n");
-        concurrent_session_runs = 1;
-      }
-      if (execution_mode == ExecutionMode::ORT_PARALLEL) {
-        fprintf(stderr, "OpenVINO doesn't support parallel executor switching to sequential executor\n");
-        sf.SetExecutionMode(ExecutionMode::ORT_SEQUENTIAL);
-      }
```

Comment:
Have you test this out? technically, I don't think there would be blocker for this with this commit. but this also bring us more concern regarding the max request queue size which contorled by nireq = 8. If the model is partitioned to multiple openvino subgraphs, and subgraph can be execute in parallel, then there will be multi-thread created for the sub-graph, and they all need to get the infer_request from the queue. hard code the max queue size to 8 may not be a good idea.

Reply:
> Have you test this out? technically, I don't think there would be blocker for this with this commit. but this also bring us more concern regarding the max request queue size which contorled by nireq = 8. If the model is partitioned to multiple openvino subgraphs, and subgraph can be execute in parallel, then there will be multi-thread created for the sub-graph, and they all need to get the infer_request from the queue. hard code the max queue size to 8 may not be a good idea.

@HectorSVC Yes I was able to test it out and I have pushed my changes where I have made **nireq value configurable**. Now, we can Override the accelerator default value of number of threads with any value at runtime. If this option is not explicitly set, default value of 8 is used during build time.